code/traffic-register/traffic-register.css

```diff
@@ -332,103 +396,332 @@ a.ang-link:link {text-decoration: none;}
+  box-shadow: 0px 0px 0px 0px gray; font-size: 1.1em; background-color: #ebebeb; color: #163f6e; text-align: center; }
+#kn-scene_748 .kn-back-link:hover { opacity: 0.9; cursor: pointer; border-color: gray; }
+
+/*Checked Out Documents*/
+#kn-scene_762 .kn-back-link { padding: 5px 10px; border-style: solid; border-width: 1px; border-color: #dddddd; border-radius: 4px; 
+  box-shadow: 0px 0px 0px 0px gray; font-size: 1.1em; background-color: #ebebeb; color: #163f6e; text-align: center; }
+#kn-scene_762 .kn-back-link:hover { opacity: 0.9; cursor: pointer; border-color: gray; }
+
+/*Unassigned Documents*/
+#kn-scene_579 .kn-back-link { padding: 5px 10px; border-style: solid; border-width: 1px; border-color: #dddddd; border-radius: 4px; 
+  box-shadow: 0px 0px 0px 0px gray; font-size: 1.1em; background-color: #ebebeb; color: #163f6e; text-align: center; }
+#kn-scene_579 .kn-back-link:hover { opacity: 0.9; cursor: pointer; border-color: gray; }
+
+/*Unconnected Regulations*/
+#kn-scene_580 .kn-back-link { padding: 5px 10px; border-style: solid; border-width: 1px; border-color: #dddddd; border-radius: 4px; 
+  box-shadow: 0px 0px 0px 0px gray; font-size: 1.1em; background-color: #ebebeb; color: #163f6e; text-align: center; }
+#kn-scene_580 .kn-back-link:hover { opacity: 0.9; cursor: pointer; border-color: gray; }
+
+/*Task Board - Draft Editor*/
+#kn-scene_658 .kn-back-link { padding: 5px 10px; border-style: solid; border-width: 1px; border-color: #dddddd; border-radius: 4px; 
+  box-shadow: 0px 0px 0px 0px gray; font-size: 1.1em; background-color: #ebebeb; color: #163f6e; text-align: center; }
+#kn-scene_658 .kn-back-link:hover { opacity: 0.9; cursor: pointer; border-color: gray; }
+
+/*Task Board - Draft Review*/
+#kn-scene_673 .kn-back-link { padding: 5px 10px; border-style: solid; border-width: 1px; border-color: #dddddd; border-radius: 4px; 
+  box-shadow: 0px 0px 0px 0px gray; font-size: 1.1em; background-color: #ebebeb; color: #163f6e; text-align: center; }
+#kn-scene_673 .kn-back-link:hover { opacity: 0.9; cursor: pointer; border-color: gray; }
+
+/*Reg Doc Details - Search*/
+#kn-scene_588 .kn-back-link { padding: 5px 10px; border-style: solid; border-width: 1px; border-color: #dddddd; border-radius: 4px; 
+  box-shadow: 0px 0px 0px 0px gray; font-size: 1.1em; background-color: #ebebeb; color: #163f6e; text-align: center; }
+#kn-scene_588 .kn-back-link:hover { opacity: 0.9; cursor: pointer; border-color: gray; }
+
+/*Reg Item Details - Search*/
+#kn-scene_581 .kn-back-link { padding: 5px 10px; border-style: solid; border-width: 1px; border-color: #dddddd; border-radius: 4px; 
+  box-shadow: 0px 0px 0px 0px gray; font-size: 1.1em; background-color: #ebebeb; color: #163f6e; text-align: center; }
+#kn-scene_581 .kn-back-link:hover { opacity: 0.9; cursor: pointer; border-color: gray; }
+
+/**************************************/
+/*** Style Details Links as Buttons ***/
```

Comment:
Does this refer to this? 
![image](https://github.com/user-attachments/assets/65edca9b-69ae-4a12-81c7-4c0ccb4dfab7)

Couldn't we just update the global styling in the app settings?

Reply:
These are links located within a Details View and need a specific styling applied to enhance usability. We link the generated PDFs and will be using them for the historical PDFs as well.
![image](https://github.com/user-attachments/assets/81bcdd5e-dae8-4c8a-8c78-27a8d603bec5)